Tribler/Core/TorrentChecker/torrent_checker.py

```diff
@@ -406,11 +422,8 @@ def _update_torrent_result(self, response):
         torrent_id = result[u'torrent_id']
         retries = result[u'tracker_check_retries']
 
-        # the result logic
-        is_good_result = seeders > 0 or leechers > 0
-
         # the status logic
-        if is_good_result:
+        if seeders > 0:
```

Comment:
@NielsZeilemaker We decided to check seeders only because the swarm can be dead if there is no leecher (although it might be alive, but the chance is relatively low).